src/mame/excellent/excellent_spr.cpp

```diff
@@ -187,15 +187,6 @@ void excellent_spr_device::aquarium_draw_sprites(screen_device &screen, bitmap_i
 			}
 		}
 	}
-
-#if 0
-	if (rotate)
-	{
-		char buf[80];
-		sprintf(buf, "sprite rotate offs %04x ?", rotate);
-		popmessage(buf);
-	}
-#endif
```

Comment:
I think this is dead code. There is no `rotate` var anywhere I can see. It dates back to the initial commit, where it still wouldn't do anything.